apps/cli/src/transforms/images.ts

```diff
@@ -111,7 +117,12 @@ export async function saveImageInStaticFolder(
     await fileObject.writeBase64(urlSource);
     file = fileObject.id;
   } else {
-    session.log.error(`⚠️  Cannot find image "${urlSource}" in ${opts?.sourceFile || filePath}`);
+    const message = `Cannot find image "${urlSource}" in ${opts?.sourceFile || filePath}`;
+    if (opts?.sourceFile) {
```

Comment:
Some of this `if/else` could be dealt with in `addWarningForFile`... maybe just `addWarning`, and if `file` isn't provided, just `log` (or even save the error as non-file-specific...). Could also add `debug` handling in there too (log, don't save)?

I guess it's a trade off between reducing lines of code and making the `addWarning` function a bit more bloated in scope.

Reply:
Fixed!